samples/excel/50-workbook/create-workbook.yaml

```diff
@@ -0,0 +1,94 @@
+order: 2
+id: excel-workbook-create-workbook
+name: Create workbook
+description: Creates new workbooks both from scratch and by copying an existing one.
```

Comment:
Consider changing description to something more like -> "Creates a new (empty) workbook and creates a new workbook by copying an existing one."